### PR TITLE
#20 standardize slack endpoint

### LINKED ISSUE #20: Standardize Slack Endpoint
This is to select an official endpoint and perform standardization.

Remember to add two or three important business flow steps.

### PR DESCRIPTION
**Pull request by agreggio-slingr**
Pull request for the topic https://github.com/slingr-stack/slack-endpoint/issues/20 created by @agreggio-slingr

## What does it do?
This ticket implements the use of slingr-helpgen, whereby it modifies existing functions, adds basic functions and step flows.

## How to test it?
Make calls from the platform for example using the console using the information provided in the documentation.

## Implementation notes
Create new version of the endpoint, it is compatible with previous versions.

## Implementation notes
Create a new tags changing the minor version and deploy to the platform.

### CODE REVIEW

**Comment by sebasmoyano** [comment]
Can you deploy changes in dev? So I can see business steps.

**Comment by agreggio-slingr** [comment]
> Can you deploy changes in dev? So I can see business steps.

Sure, I will send you the implementation link on test1 dev. Additionally, I need to add you to the test Slack group that Juanson added me to."

https://test1.idea2.io/dev/builder/#model/entities/5b2bde98e4a8324cd254cfe2/actions/6410e1b6def8070602000603

File: README.md [hunk 1]
```diff
@@ -647,3 +647,1059 @@ SLINGR is a low-code rapid application development platform that accelerates dev
...
+
+
+<details>
+    <summary>Click here to see the Customs Flow Steps</summary>
+
+<br>
+
+
+
+### Custom Flow Steps Name
```
Comment (pasaperez-work) [inline README.md]: Remember to update the Custom Flow Step documentation.

**Comment by pasaperez-work** [comment]
#approve

**Comment by sebasmoyano** [comment]
#approve

**Pull request merged**